modules/git/repo_compare.go

```diff
@@ -295,6 +295,22 @@ func (repo *Repository) GetFilesChangedBetween(base, head string) ([]string, err
 	return split, err
 }
 
+// GetCommitFilesChanged get the changed file names of the specified commit
+func (repo *Repository) GetCommitFilesChanged(commitID string) ([]string, error) {
```

Comment:
It's better to have a unit test for this function.

Reply:
Yes, I'm working on it

Reply:
added a test in 7636f7a6dbdead77ff79289d2b240a0fa1e855de